saddle.config.js

```diff
@@ -133,21 +133,21 @@ module.exports = {
     rinkeby: {
       providers: [
         {env: "PROVIDER"},
-        {file: "~/.ethereum/rinkeby-url"},                    // Load from given file with contents as the URL (e.g. https://infura.io/api-key)
-        {http: "https://rinkeby-eth.compound.finance"}
+        {http: "https://rinkeby.infura.io/v3/6c3cf934f4fb4e5a862fba01abd5aed6"}
```

Comment:
Please don't commit your own API key, better take it from process.env (update .env.example accordingly). Alternatively, you can just remove `{http: "https://rinkeby-eth.compound.finance"}` line and provide the whole url via the `PROVIDER` environment variable or via `~/.ethereum/rinkeby-url` file (e.g., `echo "https://rinkeby.infura.io/v3/6c3cf934f4fb4e5a862fba01abd5aed6" > ~/.ethereum/rinkeby-url` will make the original config just work). But we should remove compound urls from there ofc.

Reply:
changed it to refer the         {file: "~/.ethereum/rinkeby-url"},